### DISCUSSION TITLE
Welke "lijsten" van GEMMA wilen we gebruiken en hoe koppelen we die?

### DISCUSSION THREAD

**Discussion by RonaldvCortenberghe** [OpenCatalogi/.github#202] [Q&A]
In de OpenCatalogi willen we gebruik maken van GEMMA-objecten om de zoekresultaten te kunnen filteren. In eerste instantie hebben we de volgende objecten vastgesteld:

- Bedrijfsservice
- Bedrijfsfunctie
- Applicatiefuncties
- Referentiecomponenten

Zo zou je bijvoorbeeld vanuit referentiecomponent kunnen zoeken naar componenten die zo'n functionaliteit hebben

We hebben een extractie gemaakt van de GEMMA referentie architectuur en die is ingelezen. Dit is natuurlijk nu een statisch iets, m.a.w. die kunnen alleen bijgewerkt worden d.m.v. updates.

We willen die natuurlijk gekoppeld hebben zodat we die binnenhalen op commando. Er wordt nu gedacht om dit via de softwarecatalogus te koppelen. Naar mijn idee zouden we dit moeten koppelen aan de GEMMA-referentie architectuur. De softwarecatalogus is tenslotte een afnemer van de GEMMA referentie architectuur.

De vraag is dan ook; wat is de bron en kunnen we daaraan koppelen?

Verder is de vraag zijn dit de juiste objecten of moeten er meer komen?

**Comment by Makkmetp** [comment]
Op dit moment is de GEMMA-referentiearchitectuur beschikbaar via https://www.gemmaonline.nl/index.php/GEMMA_Architectuurrepository. 
Momenteel wordt gedacht om de repository aan te gaan bieden via GitHub.

**Reply by RonaldvCortenberghe** [reply]
Hallo Peter,

Is er al een beeld wanneer dit gerealiseerd is? Het liefst zou ik via een API de objecten inlezen in OpenCatalogi
